### DISCUSSION TITLE
Data on disabled or uninstalled extensions

### DISCUSSION THREAD

**Discussion by BeFunes** [microsoft/vscode-discussions#374] [Extension Development QnA] [answered]
I'm wondering if there is a way, as an extension owner, to know how many users have disabled or uninstalled my extension. 
The acquisition panel doesn't tell me anything about this, but I'm curious if VS Code has this information and there is any way to access it

**Comment by isidorn** [comment, marked as answer]
We do not have this information.
This should be a feature request against the Marketplace https://github.com/microsoft/vsmarketplace

**Reply by BeFunes** [reply]
Thanks, I will do that!
